cloudevents/spec.md

```diff
@@ -148,6 +148,15 @@ as one that is unaware of CloudEvents.
 A "batch-mode message" is one where multiple (zero or more) events are
 encoded in a single message body, according to a specific event format. Not
 all event formats or protocol bindings support batch-mode messages.
+The CloudEvents within a batch are largely independent from one another: there
+is no restriction that they have the same source, producer, content type etc.
+There are just two restrictions, however:
+
+- All CloudEvents within the same batch MUST have the same `specversion`
+  attribute value.
+- All CloudEvents within the same batch MUST be distinct, i.e. the
```

Comment:
On the 3/2 call we agreed to remove the 2nd bullet

Reply:
Done in extra commit.